sig-docs/annual-report-2021.md

```diff
@@ -4,79 +4,72 @@
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - It's updated to the best of our knowledge.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - We have a [well-documented guide](https://kubernetes.io/docs/contribute/#:~:text=Anyone%20can%20open%20an%20issue,the%20CNCF%20Contributor%20License%20Agreement.) that details how folks can get started and the various ways in which they can scale the contributor ladder.
+   - We actively assign good-first-issue labels to issues that we think early-lifecycle contributors could begin with. During KubeCons we also do introduction presentations and mentoring sessions to try and funnel contributors in to our SIG.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - [98 companies](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=SIG%20Docs&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&from=1609455600000&to=1639350000000)
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - We're in the process of formalizing the localization subproject. We aim for it to, potentially, provide more avenues for contribution. More details around leadership for the localization supbproject can be found in [this message](https://groups.google.com/a/kubernetes.io/g/dev/c/SP6weMvx3wg/m/l8LAL-OFCQAJ) sent to dev@kubernetes.io 
 
 ## Membership
 
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-- Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-- Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+- Primary slack channel member count: 1876
+- Primary mailing list member count: 453
+- Primary meeting attendee count (estimated, if needed): 10
+- Primary meeting participant count (estimated, if needed): 6
+- Unique reviewers for SIG-owned packages: 76 <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+- Unique approvers for SIG-owned packages: 93<!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
```

Comment:
Does this make sense for SIG Docs?

Reply:
Tbh, I don't think. But @parispittman would you please be able to comment if this is actually relevant for non-code owning SIGs?